laserchicken/feature_extractor/test_percentile.py

```diff
@@ -0,0 +1,36 @@
+import os
+import random
+import unittest
+
+from laserchicken import read_las
+from laserchicken.feature_extractor.percentile_feature_extractor import PercentileFeatureExtractor
+
+class TestPercentileFeatureExtractor(unittest.TestCase):
+
+    def test_percentile(self):
```

Comment:
Could you add a docstring about what scenerio is tested and what is expected. Also maybe say really short where the expected values come from.